package.json

```diff
@@ -13,7 +13,7 @@
     "Linters"
   ],
   "engines": {
-    "vscode": "^1.62.0"
+    "vscode": "^1.81.1"
```

Comment:
Is there a way to get dependabot to keep this dependency up-to-date, like it does e.g. here #167?
In any event, this should be a separate PR.